odl/solvers/iterative/statistical.py

```diff
@@ -187,22 +145,23 @@ def osmlem(op, x, data, niter=1, noise='poisson', callback=None, **kwargs):
 
     .. math::
 
-        \prod_{i=1}^M P(g_i | g_i \\text{ is } X(A_i(x)) \\text{ distributed})
+        \prod_{i=1}^M P(g_i | g_i \\text{ is } X(A_i(x)) \\text{ distributed}).
 
-    where the expectation of :math:`X(A_i(x))` satisfies
-
-    .. math::
-
-        \\mathbb{E}(X(A_i(x))) = A_i(x)
-
-    with 'poisson' noise the algorithm is given by partial updates:
+    With 'poisson' noise the algorithm is given by partial updates:
 
     .. math::
 
        x_{n + m/M} =
        \\frac{x_{n + (m - 1)/M}}{A_i^* 1} A_i^* (g_i / A_i(x_{n + (m - 1)/M}))
 
-    for :math:`m = 1, ..., M` and :math:`x_{n+1} = x_{n + M/M}`
+    for :math:`m = 1, ..., M` and :math:`x_{n+1} = x_{n + M/M}`.
+
+    The algorithm is not guaranteed to converge, but works for many practical
+    problems.
+
+    References
+    ----------
+    Natterer, F. Mathematical Methods in Image Reconstruction, section 5.3.2.
```

Comment:
Frankly we need a discussion on refs.rst at some point. I find the fact that I cant find the references while reading the doc quite irritating, and names like [ABC2016] cant be googled either.

What is the gain from having it? A centralized place for references? Avoiding duplication? Is this how others do it in general?

I would personally be in favor of scrapping it.